package.json

```diff
@@ -31,7 +31,13 @@
     "build:elements": "cd packages/elements && yarn build",
     "build:storybook": "lerna run build-storybook",
     "build:storybook:react": "cd packages/components/react-web && yarn build-storybook",
-    "build:storybook:wc": "cd packages/components/web-components && yarn build-storybook"
+    "build:storybook:wc": "cd packages/components/web-components && yarn build-storybook",
+    "clone:themes": "git clone --filter=blob:none https://github.com/compassion-global-experience/cds-tokens.git && cp -r cds-tokens/themes/ ./themes/ && rm -rf cds-tokens",
+    "build:cdsTokens": "npm run clone:themes && npm run transform && npm run resolve-tokens",
+    "transform": "npm run transform-light && npm run transform-dark",
+    "transform-light": "npx token-transformer themes tokens/_cds-light/tokens.json core/cds-core,cds-light/theme-cds-light,components/accordion,components/alert,components/background,components/badge,components/breadcrumb,components/button,components/card,components/dropdown,components/form,components/grid,components/header,components/modal,components/pagination,components/spacing,components/tab,components/table,components/toast,components/tooltip,components/type core/cds-core,cds-light/theme-cds-light",
+    "transform-dark": "npx token-transformer themes tokens/_cds-dark/tokens.json core/cds-core,cds-dark/theme-cds-dark,components/accordion,components/alert,components/background,components/badge,components/breadcrumb,components/button,components/card,components/dropdown,components/form,components/grid,components/header,components/modal,components/pagination,components/spacing,components/tab,components/table,components/toast,components/tooltip,components/type core/cds-core,cds-dark/theme-cds-dark",
```

Comment:
Why do we have to list all these items here

```
core/cds-core,cds-light/theme-cds-light,components/accordion,components/alert,components/background...
```

Can they come from config? Or at least something like `components/*` ?

Reply:
I've checked out `token-transformer` and the example there just uses `components` and not listing every component folder, can we try that?